.github/workflows/slither.yml

```diff
@@ -0,0 +1,59 @@
+name: Static-Analysis
+on:
+    push:
+        paths:
+            - 'contracts/**'
+            - '.github/workflows/slither.yml'
+            - '**.py'
+    pull_request:
+        paths:
+            - 'contracts/**'
+            - '.github/workflows/slither.yml'
+
+jobs:
+    build:
+        runs-on: ubuntu-latest
+        strategy:
+            matrix:
+                python-version: [3.6]
+                node-version: [16.x]
+
+        steps:
+            - uses: actions/checkout@v2
+            - name: Branch name
+              run: echo running on branch ${GITHUB_REF##*/}
+            - name: Use Node.js ${{ matrix.node-version }}
+              uses: actions/setup-node@v1
+              with:
+                  node-version: ${{ matrix.node-version }}
+            - run: npm install
+            - run: npm run build
+
+            - name: Set up Python ${{ matrix.python-version }}
+              uses: actions/setup-python@v2
+              with:
+                  python-version: ${{ matrix.python-version }}
+            - name: Install dependencies
+              run: |
+                  sudo snap install solc
+                  python -m pip install --upgrade pip
+                  pip install slither-analyzer solc-select
+                  solc-select install 0.6.11
+                  solc-select use 0.6.11
+            - name: Summary of static ananlysis
+              run: |
+                  echo "Running slither"
+                  slither . --print human-summary
+            - name: Contract summary of static ananlysis
+              run: |
+                  echo "Running slither"
+                  slither . --print contract-summary
+            - name: Function summary
+              run: |
+                  slither . --print function-summary
+            - name: Data dependency
+              run: |
+                  slither . --print data-dependency
```

Comment:
Same concern as above... especially this step just seems to print thousands of lines of data dependency.... is anyone going to read that? Anything more useful to be done here?

Reply:
I changed the workflow. So, I just reported the issues to do solved